src/main/drivers/accgyro/accgyro_mpu.c

```diff
@@ -49,11 +53,14 @@
 #include "drivers/accgyro/accgyro_spi_mpu6000.h"
 #include "drivers/accgyro/accgyro_spi_mpu6500.h"
 #include "drivers/accgyro/accgyro_spi_mpu9250.h"
+#include "drivers/accgyro/accgyro_imuf9001.h"
 #include "drivers/accgyro/accgyro_mpu.h"
 
-
+volatile int dmaSpiGyroDataReady = 0;
```

Comment:
this is not used, it's kind of a place holder. Maybe we should remove it for now?

Reply:
Will remove